fixieai/cli/agent/commands.py

```diff
@@ -81,6 +101,17 @@ def init_agent(handle, description, entry_point, more_info_url, public):
     current_config.public = public
     agent_config.save_config(current_config)
 
+    entry_module, _ = entry_point.split(":")
+    expected_main_path = entry_module.replace(".", "/") + ".py"
+    if not os.path.exists(expected_main_path):
+        urllib.request.urlretrieve(AGENT_TEMPLATE_URL, expected_main_path)
+        click.secho(
+            f"Initialized agent.yaml and made a template agent file at {expected_main_path}",
```

Comment:
maybe "main.py" instead of "agent" to make clear it's python?

Reply:
The default file is `main.py` (unless the user changes the entry point in the questionnaire). The `agent.yaml` holds the config